python/ray/dashboard/state_aggregator.py

```diff
@@ -341,7 +340,7 @@ async def list_nodes(self, *, option: ListApiOptions) -> ListApiResponse:
         result = list(islice(result, option.limit))
         return ListApiResponse(
             result=result,
-            total=total_nodes,
+            total=reply.total,
```

Comment:
QQ: this includes filtered numbers too. right?

Reply:
yes, a la other lists https://github.com/ray-project/ray/blob/master/python/ray/dashboard/state_aggregator.py#L264